src/app/components/VisuallyHiddenText/__snapshots__/index.test.tsx.snap

```diff
@@ -37,3 +37,23 @@ exports[`VisuallyHiddenText should render off screen text for screen readers as
   Some offscreen text
 </h1>
 `;
+
```

Comment:
I suggest to remove the snapshot and create a proper test for this.

Reply:
I think for now at least the plan is to retain the snapshots just to have a like-for-like with the legacy component behaviour but afterwards we could do a clean up to change the tests if that makes sense?

Reply:
🤔 you can update the test to follow code standard, the logic should be the same, snapshot tests don't really test anything.
Unless you mean that we want an HTML comparison with the legacy component. In which case, ship it 🚢 👍